src/hammer-vlsi/hammer_vlsi/driver.py

```diff
@@ -140,6 +140,7 @@ def load_technology(self, cache_dir: str = "") -> None:
         self.log.info("Loading technology '{0}'".format(tech_str))
         tech_opt = None  # type: Optional[hammer_tech.HammerTechnology]
         for base_path in tech_paths:
+            sys.path.append(base_path)
```

Comment:
Related change?

Reply:
This will append technology search dir as a python path for importing modules like `import asap7`